client/components/info-link/index.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+import classNames from 'classnames';
+
+export default class InfoLink extends Component {
+	static propTypes = {
+		href: PropTypes.string.isRequired
+	}
+
+	render() {
+		return (
+			<span
+				ref="infoLink"
```

Comment:
I don't think we'll need a `ref` since this component is fairly static. Do you have an idea about how we could use it in practice?